### PR TITLE
Issue 1: Added support for Stash List (opening reflog on refs/stash)

### PR DESCRIPTION
**Pull request by mbinic**
I've added a command to launch the reflog on refs/stash (Stash L`i`st), along with a toolbar icon (recolored log icon) and shortcut `Ctrl + G, I`.

### CODE REVIEW

**Comment by MattDavies** [comment]
Hey @mbinic, this is awesome - thanks so much! Looks great on the surface, I'll just test it out tomorrow then merge if all OK.

**Comment by mbinic** [comment]
Looking forward to it :)

File: TortoiseGitToolbar/Config/TortoiseGitToolbar.vsct [hunk 1]
```diff
@@ -123,35 +123,42 @@
           <ButtonText>Stash Save</ButtonText>
         </Strings>
       </Button>
-      <Button guid="guidTortoiseGitToolbarCmdSet" id="cmdSwitch" priority="0x0115" type="Button">
+      <Button guid="guidTortoiseGitToolbarCmdSet" id="cmdStashList" priority="0x0115" type="Button">
+        <Parent guid="guidTortoiseGitToolbarCmdSet" id="ToolbarGroup" />
+        <Icon guid="guidImages" id="iconStashList" />
+        <Strings>
+          <ButtonText>Stash List</ButtonText>
```
Comment (MattDavies) [inline TortoiseGitToolbar/Config/TortoiseGitToolbar.vsct]: I think what you have is fine for now. In the future, we can always run Git first to test if the stash is empty and show a dialog if needed.

**Comment by MattDavies** [comment]
Released as v0.996.1, thanks again!

**Pull request merged**